content/de/includes/partner-style.css

```diff
@@ -0,0 +1,202 @@
+/*  SECTIONS  */
```

Comment:
Unless specific styling is added, it is better to have to this file removed. Then `content/en/includes/partner-style.css` will be used.